tests/backtrace/host/host.cpp

```diff
@@ -38,7 +38,6 @@ int main(int argc, const char* argv[])
             "ecall_test",
             "oe_handle_call_enclave_function",
             "_handle_ecall",
-            "__oe_handle_main",
```

Comment:
Did you see this change after switching to clang-11?

Reply:
@asvrada do you remember why this change was made?

Reply:
For some reason after upgrading to clang-11, `__oe_handle_main` is gone.

Reply:
Can you create an issue to investigate this?

Reply:
https://github.com/openenclave/openenclave/issues/4704